### ISSUE TITLE
No alt text on any icons (Issue 122 from the accessibility list)

### ISSUE THREAD

**Issue by BatJan** [umbraco/Umbraco-CMS#5904]
The icons should be hidden for screen readers using the aria-hidden="true" attribute and the <a> tag should be refactored to a <button> and the tabindex="-1" should be removed so the buttons are possible to navigate using a keyboard.

List of issues: https://github.com/umbraco/Umbraco-CMS/issues/5277

I'll do a PR shortly.
